registry/server/common/services/validateRequest.ts

```diff
@@ -26,8 +26,8 @@ const validateRequestFactory = (validationConfig: ValidationConfig[]) => async (
             async ({schema, selector}) => schema.validateAsync(selector(req)),
             validationConfig
         ));
-    } catch (error) {
-        res.status(422).send(preProcessErrorResponse(error));
+    } catch (e) {
```

Comment:
I have one notice here
I want you to pay attention to consistent because of as I see you wrote `e` in the second file you wrote `err` also we already have `error`
So I suggest to follow a naming convention in this case